packages/subgraph/package.json

```diff
@@ -1,27 +1,30 @@
 {
+    "type": "module",
     "private": true,
     "name": "@carrot-kpi/subgraph",
     "version": "0.10.0",
     "license": "GPL-3.0-or-later",
     "scripts": {
-        "gen-networks-files": "ts-node scripts/generate-networks-files.ts",
+        "gen-networks-files": "ts-node-esm --project tsconfig.scripts.json scripts/generate-networks-files.ts",
         "codegen": "graph codegen && yarn gen-networks-files",
         "clean": "rimraf generated build && rimraf generated src/gen",
         "build": "yarn clean && yarn codegen && graph build",
         "deploy:sepolia": "yarn gen-networks-files && graph deploy --product hosted-service --network sepolia carrot-kpi/carrot-sepolia",
         "deploy:gnosis": "yarn gen-networks-files && graph deploy --product hosted-service --network gnosis carrot-kpi/carrot-gnosis",
+        "deploy:mumbai": "yarn gen-networks-files && graph deploy --product hosted-service --network mumbai carrot-kpi/carrot-mumbai",
         "create-local": "graph create --node http://localhost:8020/ carrot-kpi/carrot-kpi-local",
         "remove-local": "graph remove --node http://localhost:8020/ carrot-kpi/carrot-kpi-local",
         "deploy-local": "graph deploy --node http://localhost:8020/ --ipfs http://localhost:5001 carrot-kpi/carrot-kpi-local",
         "test": "echo 'Make Matchstick work...'"
     },
     "devDependencies": {
-        "@carrot-kpi/contracts": "^0.8.0",
-        "@graphprotocol/graph-cli": "0.59.0",
+        "@carrot-kpi/contracts": "^0.9.0",
+        "@graphprotocol/graph-cli": "0.60.0",
         "@graphprotocol/graph-ts": "0.31.0",
         "eslint-config-custom": "*",
-        "matchstick-as": "0.5.2",
+        "matchstick-as": "0.6.0",
         "rimraf": "^5.0.5",
-        "ts-node": "^10.9.1"
+        "ts-node": "^10.9.1",
+        "tsconfig": "*"
```

Comment:
Is this desired? To have any version?

Reply:
`tsconfig` is a local dependency internal to the monorepo, so with `yarn` that's how you specify it.